Controller/Adminhtml/YotpoController/YotpoController.php

```diff
@@ -50,13 +50,14 @@ public function execute()
         return;
       }
       $offset = 0;
-      $orderStatuses = $this->_config->getCostumeOrderStatus();
+      $orderStatuses = $this->_config->getCustomOrderStatus();
```

Comment:
No it was defined just as a placeholder if you review in Block/Config.php

>     public function getCostumeOrderStatus()
>     {
>         null;
>     }

Further store id is passed to the status as well now in 7566083